boa/contracts/vyper/vyper_contract.py

```diff
@@ -810,7 +810,7 @@ def unoptimized_bytecode(self):
 
     @cached_property
     def unoptimized_ir(self):
-        settings = self.compiler_data.settings.copy()
```

Comment:
what's the issue here?

Reply:
```
    @cached_property
    def unoptimized_ir(self):
>       settings = self.compiler_data.settings.copy()
E       AttributeError: 'Settings' object has no attribute 'copy'
```

Reply:
err -- then we should use `copy.copy()`

Reply:
ah cool: https://github.com/vyperlang/titanoboa/pull/220/commits/b431d4e7eb7b6e8ef286e90bf9728af9d0544707